File: cpp/47c_1843A_SashaAndArrayColoring.cpp
#include<bits/stdc++.h>

int solve(std::vector<int> arr, int n){
    int cnt = 0;
    for(int i = 0; i < (n / 2); i++){
        cnt += (arr[n - 1 - i ] - arr[i]);
        //std::cout<<cnt<<" "<<arr[n - 1 - i]<<" "<<arr[i]<<"\n";
    }
    return cnt;
}

int main(){
    int t, n, temp;
    std::vector<int> ans;
    std::cin>>t;
    for (int i = 0; i < t; i++){
        std::cin>>n;
        std::vector<int> arr;
        for(int j = 0; j < n; j++){
            std::cin>>temp;
            arr.push_back(temp);
        }
        sort(arr.begin(), arr.end());
        ans.push_back(solve(arr, n));
    }
    for (auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/47c_1843A_SashaAndArrayColoring.py
def func(arr, n):
    cnt = 0
    for i in range(n//2):
        cnt += (arr[n-1-i] - arr[i])
    return cnt

def main():
    ans = []
    t = int(input())
    for _ in range(t):
        n = int(input())
        arr = input().split()
        for i in range(n):
            arr[i] = int(arr[i])
        arr.sort()
        ans.append(func(arr, n))
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()